ivy/functional/backends/jax/activations.py

```diff
@@ -85,11 +77,7 @@ def softsign(x: JaxArray, /, *, out: Optional[JaxArray] = None) -> JaxArray:
     return jax.nn.soft_sign(x)
 
 
-def log_softmax(
-    x: JaxArray, /, *, axis: Optional[int] = None, out: Optional[JaxArray] = None
-):
-    if axis is None:
-        axis = -1
+def log_softmax(x: JaxArray, /, *, axis: int = -1, out: Optional[JaxArray] = None):
     return jax.nn.log_softmax(x, axis)
 
 
```

Comment:
The changes to the `axis` of `ivy.softmax` seem to be causing failures in `test_tensorflow_softmax`, `test_tensorflow_Softmax` and `test_tensorflow_sparse_categorical_crossentropy`.
The changes to the `axis` of `ivy.log_softmax` also seem to be causing failures in `test_tensorflow_LogSoftmax`.